Training/1.0 Introduction.md

```diff
@@ -12,37 +12,53 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+list = ["one",2,"3"]
+string = "Hello World!"
+integer = 1024
+boolean = True
+dictionary = {"one":1,"two":2}
+tuple = (1,2,3,4)
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+a = ["I", "am", "a", "list"]
+b = [1,2,3,4]
+c = (1,2,3,4)
 ```
 
 ### Task 3: 
 
-We are also interested in becoming good software developers so we will need to use conditional, loops. Write an statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
+We are also interested in becoming good software developers so we will need to use conditional statements. Write a statement where the summation of a variable x+1 will be calculated if the value of x is greater than 2. 
 
 ```
-
+x = 5
+if x > 2:
+  print(x++)
 ```
 
-Now incorporate a for loop to calculate the value of n(x+1) for n repetitions where n=3. Store each value of the for loop in an array named y.  
+Now incorporate a for loop to calculate the value of n(x+i) for n repetitions where n=3. Store each value of the for loop in an array named y.  
 
 ```
-
+y = []
+for i in range(n):
+  y.append(n*(x+i))
 ```
 
 ### Task 4: 
 
 Finally we will have a look at creating functions. Functions allow us to compute processes much faster without having to repeat lines of code. Using your code from Task 3, create a function called 'my_cool_function', which takes in the value x and will compute the n(x+1) if x is greater than 2 and for n repetitions where values will be stored in an array named y. Your function will return y. 
 
 ```
-
+def my_cool_function(x):
+  y = []
+  if x > 2:
+    n*x++
+  for i in range(n):
+    y.append(n*(x+i))
```

Comment:
Correct function definition but missing return statement. At the end you need to add:
`return y`
Also see array creation with numpy in previous comment.